lib/APIDigester/ModuleAnalyzerNodes.cpp

```diff
@@ -1662,12 +1659,6 @@ SwiftDeclCollector::constructTypeNode(Type T, TypeInitInfo Info) {
 
   SDKNode* Root = SDKNodeInitInfo(Ctx, T, Info).createSDKNode(SDKNodeKind::TypeNominal);
 
-  // Keep paren type as a stand-alone level.
-  if (auto *PT = dyn_cast<ParenType>(T.getPointer())) {
-    Root->addChild(constructTypeNode(PT->getSinglyDesugaredType()));
-    return Root;
-  }
-
```

Comment:
@nkcsgexi Any concerns with removing ParenType from the api-digester? This should only affect when it runs in non-ABI mode